tests/loader/test_hilti_loader.py

```diff
@@ -30,32 +30,17 @@ def test_length(self) -> None:
 
     def test_map_to_rig_idx(self) -> None:
         for i in range(0, 5, 1):
-            self.assertEqual(self.loader.map_image_idx_to_rig(i), 0)
+            self.assertEqual(self.loader.rig_from_image(i), 0)
```

Comment:
nit: change function name

Reply:
I think function names that have "to" in them are a lot less clear than those with "from".

```
A = A_from_B(B)
```